cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/pkg/errors"
 	"k8s.io/klog"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why we are renaming this import?

Reply:
goimports do does it automatically if i recall correctly.
there was a discussion about this in k/k but overall it's fine, as long as we don't sent huge PRs that modify all of these at once.